os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -415,6 +423,20 @@ int8_t cmd_wifi_ap(trwifi_softap_config_s *softap_config)
 	return ret;
 }
 
+void wlan_store_connect_info(void)
+{
+	u8 *ifname[1] = {(u8*)WLAN0_NAME};
+	rtw_wifi_setting_t setting;
+
+	if(wifi_get_setting((const char*)ifname[0],&setting) || setting.mode == RTW_MODE_AP){

```

Comment:
If `wifi_get_setting()` works well, then this if condition will be passed regardless of checking `setting.mode`.
Checking `setting.mode` is working only when `wifi_get_setting()` is failed.
But `setting.mode` cannot be `RTW_MODE_AP` without `wifi_get_setting()`.
So I think checking `setting.mode` in this condition is not necessary.
If needed, it is better to check after this if condition.

Reply:
Modified according to the suggestion.